docs/running-your-own-experiment/running_an_experiment.md

```diff
@@ -56,6 +56,10 @@ trials: 5
 # 1 day = 24 * 60 * 60 = 86400
 max_total_time: 86400
 
+# The docker registry for your fuzzbench experiments.
+# In default, we use gcr.io/fuzzbench if using Google Cloud.
```

Comment:
I think what we mean to say here is something like:
```
# If you use Google Cloud, this can be gcr.io/$PROJECT_NAME
```
Also, i think it should be 

docker_registry: gcr.io/$PROJECT_NAME
instead of 
docker_registry: gcr.io/$FUZZBENCH

no?

Reply:
Not very sure but I guess you are right. @jonathanmetzman wdyt?